### ISSUE TITLE
Issue installing imager (X11 not found) Ubuntu 18.04

### ISSUE THREAD

**Issue by yabmtm** [dahtah/imager#107]
I'm new to R but I've been having issues installing the package on Ubuntu 18.04.

```> install.packages('imager')
--- Please select a CRAN mirror for use in this session ---
trying URL 'https://cran.uib.no/src/contrib/imager_0.41.2.tar.gz'
Content type 'application/x-gzip' length 5699693 bytes (5.4 MB)
==================================================
downloaded 5.4 MB

* installing *source* package ‘imager’ ...
** package ‘imager’ successfully unpacked and MD5 sums checked
** using staged installation
checking for gcc... /home/tug27224/packages/miniconda3/envs/msm/bin/x86_64-conda_cos6-linux-gnu-cc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether /home/tug27224/packages/miniconda3/envs/msm/bin/x86_64-conda_cos6-linux-gnu-cc accepts -g... yes
checking for /home/tug27224/packages/miniconda3/envs/msm/bin/x86_64-conda_cos6-linux-gnu-cc option to accept ISO C89... none needed
checking how to run the C preprocessor... /home/tug27224/packages/miniconda3/envs/msm/bin/x86_64-conda_cos6-linux-gnu-cpp
checking for X... no
configure: error: X11 not found but required, configure aborted.
ERROR: configuration failed for package ‘imager’
* removing ‘/home/tug27224/packages/miniconda3/envs/msm/lib/R/library/imager’

The downloaded source packages are in
        ‘/tmp/RtmprRNDNV/downloaded_packages’
Updating HTML index of packages in '.Library'
Making 'packages.html' ... done
Warning message:
In install.packages("imager") :
  installation of package ‘imager’ had non-zero exit status```

I have tried everything I can think of, e.g.:
```sudo apt-get install xorg 
sudo apt-get install libx11-dev 
sudo apt-get install libglu1-mesa-dev ```
and more. Although strangely enough, I can import rgl just fine which usually depends on X11 as well.

**Comment by konradmayer** [comment]
I just installed `imager` on Ubuntu 18.04.3. I had to install `libcairo2-dev` in addition to the suggested packages. Also there is a typo in the readme - `libx11-dev` is written with a lower case "x". Opening a PR seems like overkill for this, but should I open a separate issue?

**Comment by salim-b** [comment]
> Also there is a typo in the readme - `libx11-dev` is written with a lower case "x". Opening a PR seems like overkill for this, but should I open a separate issue?

See #129
